packages/clerk-js/src/utils/url.ts

```diff
@@ -210,6 +210,18 @@ export const trimTrailingSlash = (path: string): string => {
   return (path || '').replace(/\/+$/, '');
 };
 
+/**
+ * trimLeadingSlash(path: string): string
+ *
+ * Strips the leading slashes from a string
+ *
+ * @returns {string} Returns the string without leading slashes
+ * @param path
+ */
+export const trimLeadingSlash = (path: string): string => {
```

Comment:
Can we add a couple of tests for this util?

Reply:
Yeap, thanks for the reminder